### PR TITLE
feat: add tasks page

### PR DESCRIPTION
**Pull request by manuel-rw**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [x] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [x] Pull request targets ``dev`` branch
- [x] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

image

### CODE REVIEW

**Comment by manuel-rw** [comment]
@Meierschlumpf can you suggest an implementation for listing the tasks and triggering them? I was thinking about publishing redis and pushing real time updates before and after a task started / completed

**Comment by Meierschlumpf** [comment]
Has been added in branch `cron-job-status-runner`, will be rebased into this branch soon

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]

**Review by Meierschlumpf — APPROVED** [review]
Looks good, your pnpm lock is not up to date

**Comment by manuel-rw** [comment]
dismissed review due to rebase

File: apps/nextjs/src/app/[locale]/manage/tools/tasks/_components/jobs-list.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <Stack>
+      {jobs.map((job) => (
+        <Card key={job.job.name}>
+          <Group justify={"space-between"} gap={"md"}>
+            <Stack gap={0}>
+              <Group>
+                <Text>{t(`${job.job.name}.label`)}</Text>
+                {job.status?.status === "idle" && <Badge variant="default">Idle</Badge>}
+                {job.status?.status === "running" && <Badge color="green">Running</Badge>}
+                {job.status?.lastExecutionStatus === "error" && <Badge color="red">Error</Badge>}
```
Comment (Meierschlumpf) [inline apps/nextjs/src/app/[locale]/manage/tools/tasks/_components/jobs-list.tsx:63]: Translations missing

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]

File: packages/translation/src/lang/en.ts [hunk 2]
```diff
@@ -1508,4 +1528,20 @@ export default {
...
+        label: "Simple App Ping",
+      },
+      iconsUpdater: {
+        label: "Icons Updater",
+      },
+      analytics: {
+        label: "Analytics",
+      },
+    },
+  },
```
[the remark this answers is not in the document]
Comment (Meierschlumpf) [reply packages/translation/src/lang/en.ts:1551]: It seems like this stuff is back again :joy:

File: apps/nextjs/src/app/[locale]/manage/tools/tasks/_components/jobs-list.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import React from "react";
+import { ActionIcon, Badge, Card, Group, Stack, Text } from "@mantine/core";
+import { useListState } from "@mantine/hooks";
+import { IconPlayerPlay } from "@tabler/icons-react";
+
+import type { RouterOutputs } from "@homarr/api";
+import { clientApi } from "@homarr/api/client";
+import { useTimeAgo } from "@homarr/common";
+import type { TaskStatus } from "@homarr/cron-job-status";
+import { TranslationKeys } from "@homarr/translation";
```
Comment (Meierschlumpf) [inline apps/nextjs/src/app/[locale]/manage/tools/tasks/_components/jobs-list.tsx:12]: When you are already addressing something, you can also add type to this import

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]
Would be happy if you don't force push as it is a lot easier to review only those changes made and not again taking a look on the whole file content.

**Review by Meierschlumpf — APPROVED** [review]

**Comment by manuel-rw** [comment]
> Would be happy if you don't force push as it is a lot easier to review only those changes made and not again taking a look on the whole file content.

Sorry, will do in the future 😅👍

**Pull request merged**